public/app/plugins/panel/debug/plugin.json

```diff
@@ -14,6 +14,7 @@
     "logos": {
       "small": "img/icn-debug.svg",
       "large": "img/icn-debug.svg"
-    }
+    },
+    "links": [{ "name": "Raise issue", "url": "https://github.com/grafana/grafana/issues/new" }]
```

Comment:
is this even documented anywhere? how do you use this panel?

Reply:
Not really relevant to this changes I think?

Reply:
more calling it out as odd